plugin.py

```diff
@@ -351,6 +351,8 @@ def selected_unit_test_method_names(self, view):
             for i, area in enumerate(function_areas):
                 if not area.a <= region.a <= area.b:
                     continue
+                if not i in function_regions and not area.intersects(function_regions[i]):
```

Comment:
> You can probably loop through the areas and instead of looking up the region by area index, find the entity.name.function selector within the area.

I am not sure how to find a selector within a region. What about something like this?